packages/smooth_app/pubspec.yaml

```diff
@@ -84,6 +84,9 @@ dependencies:
     path: ../data_importer
   share_plus: ^4.0.8
 
+  # Logs
+  fimber: ^0.6.6
```

Comment:
We don't need this comment here, or should put all deps in groups

Reply:
Initially there were two dependencies. Removing the comment